discord/bot.js

```diff
@@ -1129,6 +1132,10 @@ client.on("interactionCreate", async (interaction) => {
                     break;
                 }
                 case "valstatus": {
+                    if (!valorantUser) return await interaction.reply({
+                        embeds: [basicEmbed(s(interaction).error.NOT_REGISTERED)],
+                        ephemeral: true
+                    });
```

Comment:
Is there a reason someone should need an account to use /valstatus?

Reply:
It's needed considering the command goes based off the users region, so if they don't have an account it will just not work.